packages/app/src/app/overmind/namespaces/editor/actions.ts

```diff
@@ -120,11 +121,7 @@ export const sandboxChanged: AsyncAction<{ id: string }> = withLoadApp<{
     state.editor.modulesByPath = fs;
   });
 
-  if (
-    sandbox.featureFlags &&
-    sandbox.featureFlags.containerLsp &&
-    !sandbox.owned
-  ) {
+  if (CONTAINER_LSP && !sandbox.owned) {
```

Comment:
If we use local storage the flag will always exist so you need to check if it's true aswell

Reply:
Oops my bad 🙈